test/HardwareObjectsMockup.xml/beamline-setup.xml

```diff
@@ -86,4 +86,7 @@
     <osc_range>720</osc_range>
     <number_of_images>10000</number_of_images>
   </acquisition_limit_values>
+
+  <click_centring_num_clicks>1</click_centring_num_clicks>
```

Comment:
should not be this 3?

Reply:
Not for mockups :), since you re using 1 click centring.

Reply:
my headache!!